### ISSUE TITLE
Option to edit Solomon field

### ISSUE THREAD

**Issue by alrasch** [alexbonjour/rules-of-thumb#36]
Hi,

During my play, I prefer to bump Solomon first, and adjust everything after that.

Right now, that involves upgrading Solomon in-game, and then trying different Siy-numbers until the Solomon-level matches my in-game Solomon level.

I'd very much like to be able to edit the Solomon field, and recieve a recommended Siyalatas level from that.

**Comment by ShiroiKa** [comment]
I agree, editing any one field should be able to math the other fields to a greater or lesser extent.
I also think just flat having a spot to input (whole number, scientific, or abbreviated) your current number of souls should map out where they all go, so you don't have any guessing at all as to where Sya should stop before you have to check if you've spent too many souls yet.

Also, thanks again for this awesome program Alex!
